CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+
+A good bug report is one that makes it easy for everyone to understand what you were trying to do and what went wrong. Provide as much context as possible so we can try to recreate the issue.
+
+## Contributions 
+
+Contributing to the Open Liberty guides is simple. All you need to do is create your own branch to include your contribution(s). If you want to make updates to existing guides, create a new branch and open a pull request to the `qa` (quality assurance) branch.
+
+### Text or Content Contributions
+
+If you are contributing text of content changes via a pull request and you certify the [Developer Certificate of Origin (DCO)](https://developercertificate.org)), add a line to the end of the Git commit message to sign your work:
+
+```
+Signed-off-by: Jane Williams <jane.williams@gmail.com>
+```
+
+The sign-off is just a line at the end of the commit message that certifies that you wrote it or otherwise have the right to pass it on as an open source patch.
+
+Use your real name when you sign. We can't accept pseudonyms or anonymous contributions. To open a pull request to the `qa` branch, follow the official [git documentation](https://git-scm.com/docs/git-request-pull). If reviewers agree with your change, your change will be merged to the `master` branch for publishing.
+
+### Code Contributions
+
+If you are contributing trivial changes such as typos, redundant spaces, minor formatting and spelling errors, you must follow the above instructions under [Text or Content Contributions](https://github.com/OpenLiberty/guide-maven-intro/blob/dco-file/CONTRIBUTING.md#text-or-content-contribution).
+
+If you are contributing code changes via a pull request for anything except trivial changes, you must signoff on the [Individual Contributor License Agreement](https://github.com/OpenLiberty/open-liberty/blob/master/cla/open-liberty-cla-individual.pdf) If you are doing this as part of your job you may also wish to get your employer to sign a CCLA [Corporate Contributor License Agreement](https://github.com/OpenLiberty/open-liberty/blob/master/cla/open-liberty-cla-corporate.pdf). Instructions how to sign and submit these agreements are located at the top of each document. 
+
+After we obtain the signed CLA, you are welcome to open a pull request against the `qa` branch, and the team will be notified for review. We ask you follow these steps through the submission process.
+
+1. Open PR's against the `qa` branch, as we ensure changes pass our series of verification buckets before pushing to master.
+2. A "CLA signed" label will be added.
+3. A team of "reviewers" will be notified and will review your change. If approved, a full integration build will be invoked.
```

Comment:
- [ ] I don't think we do "...a full integration build will be invoked." It is for OL build.

Reply:
will be addressed by gkwan-ibm-patch-1 branch